modules/ding_library/ding_library.features.taxonomy.inc

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-937

Reply:
This case does not describe this change.

Reply:
Provides a default taxonomy vocabulary for the library type dropdown field options - https://oplevbrondby.dk/admin/structure/taxonomy/library_type
![image](https://user-images.githubusercontent.com/574498/98229562-bd698f80-1f62-11eb-9c3c-1a12fb9ae891.png)